skimage/util/tests/test_shape.py

```diff
@@ -143,14 +143,6 @@ def test_view_as_windows_with_skip():
     assert_equal(C.shape, (1, 1, 2, 2))
 
 
-def test_views_non_contiguous():
-    A = np.arange(16).reshape((4, 4))
-    A = A[::2, :]
-
-    assert_warns(RuntimeWarning, view_as_blocks, A, (2, 2))
-    assert_warns(RuntimeWarning, view_as_windows, A, (2, 2))
```

Comment:
@sciunto rather than removing this test, can you make sure it works correctly? Other than that, LGTM!

Reply:
you mean by adding a test? (then how?) or just checking manually?

Reply:
No no, I just meant check that it runs without error. Although checking manually is not too hard:

```python
image = np.arange(8)
image_non_contiguous = image[::2]
expected = np.array([[0, 2], [2, 4], [4, 6]])
assert_equal(view_as_windows(image_non_contiguous, (2,)), expected)
```

For a clean 2D test, just spot-check one of the values I guess:

```python
image = np.zeros((6, 6))
image[::2, ::2] = np.arange(9).reshape((3, 3))
assert_equal(view_as_windows(image[::2, ::2], (2, 2))[1, 1], [[4, 5], [7, 8]])
```